src/Components/GEOSldas_GridComp/GEOS_LdasGridComp.F90

```diff
@@ -620,22 +620,43 @@ subroutine Initialize(gc, import, export, clock, rc)
        close(10)
        call io_grid_def_type('w', logunit, tile_grid_f, 'tile_grid_f')
 
+       ! get a grid for perturbations and EnKF:
+       !
+       !  tile grid                 !  pert grid
+       !  (defines tile space)      !  (used for perturbations and as "hash" grid in EnKF analysis)
+       ! ===========================================================================================================
+       !  lat/lon                   !  same as tile_grid (i.e., lat/lon)
+       ! -----------------------------------------------------------------------------------------------------------
+       !  EASEv[X]                  !  same as tile_grid (i.e., EASE)
+       ! -----------------------------------------------------------------------------------------------------------
+       !  cube-sphere               !  lat/lon grid of resolution similar to that of (cube-sphere) tile_grid
+       
        pert_grid_g = get_pert_grid(tile_grid_g)
-       pert_grid_f = get_pert_grid(tile_grid_f)
-
-       if(trim(grid_type) == "Cubed-Sphere" ) then
+       ! pert_grid_f = get_pert_grid(tile_grid_f) -- SEEMS REDUNDANT, OVERWRITTEN BELOW: pert_grid_f = get_minExtent_grid()
```

Comment:
@weiyuan-jiang: I assume this is not needed, but I'm not 100% sure.  It's possible that some fields of pert_grid_f that are set here are not filled in function get_minExtent_grid() below.  This may need further edits.  If so, then we should try to clean this up further, rather than just uncommenting this line again.